kadmin/get.c

```diff
@@ -61,6 +61,7 @@ static struct field_name {
     { "fail_auth_count", KADM5_FAIL_AUTH_COUNT, 0, 0, "Fail count", "Failed login count", RTBL_ALIGN_RIGHT },
     { "policy", KADM5_POLICY, 0, 0, "Policy", "Policy", 0 },
     { "keytypes", KADM5_KEY_DATA, 0, KADM5_PRINCIPAL | KADM5_KVNO, "Keytypes", "Keytypes", 0 },
+    { "srvrkeytypes", KADM5_TL_DATA, KRB5_TL_ETYPES, 0, "Server keytypes", "Supported keytypes (servers)", 0 },
```

Comment:
fail_auth_count, pkinit-acl – perhaps server-keytypes?

Reply:
Fair!

Reply:
Should it be `server_keytypes`?  Older items use `_`s, while just one (newer) item uses `-`.